.github/workflows/test-python.yml

```diff
@@ -22,5 +22,6 @@ jobs:
         python-version: 3.8
     - shell: bash -l {0}
       run: |
+        pip install pytest-cov
```

Comment:
should we put this in the conda env?

Reply:
i think it might clutter the conda-env, since we only use it in CI. But i may be able to be persuaded otherwise